EMS/core-bundle/src/Resources/views/user/permissions.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends '@EMSCore/base.html.twig' %}{% trans_default_domain 'EMSCoreBundle' %}
+
+{% block pagetitle %}Permissions{% endblock %}
+
+{% block body %}
+    <style>
+        .table {
+            border-collapse: collapse;
+            width: 100%;
+        }
+        .table th, .table td {
+            border: 1px solid #ddd;
+            padding: 8px;
+            text-align: left;
+        }
+        .table tr:nth-child(even) {
+            background-color: #ffffff;
+        }
+        .table tr:nth-child(odd) {
+            background-color: #f2f2f2;
+        }
+        .table th {
+            background-color: dodgerblue;
+            color: white;
+        }
+    </style>
+
+
+    <table class="table table-hover">
+        <thead>
+        <tr>
+            <th>Content type</th>
+            <th>View</th>
+            <th>Create</th>
+            <th>Edit</th>
+            <th>Publish</th>
+            <th>Delete</th>
+            <th>Trash</th>
+            <th>Archive</th>
+            <th>Show link create</th>
+            <th>Show link search</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for contentType in contentTypes %}
+            <tr>
+                <td>{{ contentType.name }}</td>
+                <td>{{ contentType.roles.view|replace({'ROLE_': ''}) }}</td>
+                <td>{{ contentType.roles.create|replace({'ROLE_': ''}) }}</td>
+                <td>{{ contentType.roles.edit|replace({'ROLE_': ''}) }}</td>
+                <td>{{ contentType.roles.publish|replace({'ROLE_': ''}) }}</td>
+                <td>{{ contentType.roles.delete|replace({'ROLE_': ''}) }}</td>
+                <td>{{ contentType.roles.trash|replace({'ROLE_': ''}) }}</td>
+                <td>{{ contentType.roles.archive|replace({'ROLE_': ''}) }}</td>
+                <td>{{ contentType.roles.show_link_create|replace({'ROLE_': ''}) }}</td>
+                <td>{{ contentType.roles.show_link_search|replace({'ROLE_': ''}) }}</td>
```

Comment:
Use translation keys The roles are translated in the emsco-forms domain and in the EMSCoreBundle domain `{{ contentType.roles.show_link_create|trans }}` should do the trick